<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoForm</class>
 <widget class="QWidget" name="MemoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QLineEdit" name="linePath">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushFix">
     <property name="text">
      <string>Fix</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="NoteEdit" name="noteEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NoteEdit</class>
   <extends>QTextEdit</extends>
   <header>../src/gui/editor/noteedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
